erpnext/accounts/doctype/sales_invoice/sales_invoice.js

```diff
@@ -323,17 +323,13 @@ erpnext.accounts.SalesInvoiceController = erpnext.selling.SellingController.exte
 		this.frm.refresh_fields();
 	},
 
-	write_off_outstanding_amount_automatically: function() {
-		if(cint(this.frm.doc.write_off_outstanding_amount_automatically)) {
+	write_off_outstanding_amount_automatically() {
```

Comment:
```suggestion
	write_off_outstanding_amount_automatically () {
```

Reply:
Sorry, I added that by mistake. Ideally, a space comes after the function keyword, but not after name. [Example](https://github.com/frappe/frappe/blob/develop/frappe/public/js/frappe/form/form.js#L110)

Reply:
I'll undo it.